src/UI/Badge.elm

```diff
@@ -1,55 +1,58 @@
-module UI.Badge exposing (dark, error, light, primary, success, warning)
+module UI.Badge exposing (Badge, badge, toEl)
```

Comment:
Why did you remove the badge variations? they are there in the design system, check it [here](https://www.sketch.com/s/a252928f-582c-4f38-a5ee-14d60a2a3067/a/G33Qz8)

Reply:
WIP